zach.js

```diff
@@ -1,113 +1,55 @@
-
- //Airbnb API
- const optionsA = {
-
+//Airbnb API
 
 const optionsA = {
-
-    method: 'GET',
-	headers: {
-        'X-RapidAPI-Key': 'e22f94d494msh38033a92e3d0750p1548a6jsn35a14fbbd1d8',
-		'X-RapidAPI-Host': 'airbnb13.p.rapidapi.com'
-	}
+  method: "GET",
+  headers: {
+    "X-RapidAPI-Key": "e22f94d494msh38033a92e3d0750p1548a6jsn35a14fbbd1d8",
+    "X-RapidAPI-Host": "airbnb13.p.rapidapi.com",
+  },
 };
-
-  
-
-// //(console.log(city))
-// fetch('https://airbnb13.p.rapidapi.com/search-location?location=' + city + '&checkin=2023-04-15&checkout=2023-04-20&adults=1&children=0&infants=0&page=1&currency=USD', optionsA)
-// 	.then(response => response.json())
-// 	//.then(response => console.log(response))
-//     //.then (console.log(city))
-
-//           .then((response) =>{
-    //             console.log('response',response);
-    
-    
-    //             const {deeplink}= response.deeplink[0];
-    //             const {Airb}= response.results[0].name;
-    //             const {rating}= response.results[0].rating;
-    //             //Query all the weather data and pass it to the app
-    
-    //             document.querySelector("#deeplink").innerText = deeplink;
-    //             document.querySelector("#name").innerText = Airb;
-//             document.querySelector("#rating").innerText = rating;
-//           });
-
```

Comment:
I also got rid of the excess script that was commented out to make it look cleaner.